VocaDbWeb/wwwroot/Content/Styles/ExtLinks.less

```
@@ -493,3 +511,73 @@ a.extLink[href^="https://vocaloid.fandom.com/wiki/"]
 {
 	background: url('/Content/ExtIcons/vocawiki.png') no-repeat left 50%;
 }
+
+a.extLink[href^="http://linkco.re/"], a.extLink[href^="https://linkco.re/"]
+{
+	background: url('/Content/ExtIcons/linkcore.png');
+}
+
+a.extLink[href^="http://s.awa.fm/"], a.extLink[href^="https://s.awa.fm/"]
+{
+	background: url('/Content/ExtIcons/awa.png');
+}
+
+a.extLink[href^="http://recochoku.jp/"], a.extLink[href^="https://recochoku.jp/"]
+{
+	background: url('/Content/ExtIcons/recochoku.png');
+}
+
+a.extLink[href^="https://www.deezer.com/"], a.extLink[href^="http://www.deezer.com/"]
+{
+	background: url('/Content/ExtIcons/deezer.png');
+}
+
+a.extLink[href^="https://commons.nicovideo.jp/"], a.extLink[href^="http://commons.nicovideo.jp/"]
+{
+	background: url('/Content/ExtIcons/nicommons.png');
+}
+
+a.extLink[href^="https://seiga.nicovideo.jp/"], a.extLink[href^="http://seiga.nicovideo.jp/"]
+{
+	background: url('/Content/ExtIcons/seiga.png');
+}
+
+a.extLink[href^="http://music.dmkt-sp.jp/"], a.extLink[href^="https://music.dmkt-sp.jp/"]
+{
+	background: url('/Content/ExtIcons/dmusic.png');
+}
+
+a.extLink[href^="http://linktr.ee/"], a.extLink[href^="https://linktr.ee/"]
+{
+	background: url('/Content/ExtIcons/linktree.png');
+}
+
+a.extLink[href^="http://musicstore.auone.jp/"], a.extLink[href^="https://musicstore.auone.jp/"]
+{
+	background: url('/Content/ExtIcons/auone.png');
+}
+
+a.extLink[href^="http://nex-tone.link/"], a.extLink[href^="https://nex-tone.link/"]
+{
+	background: url('/Content/ExtIcons/nextone.png');
+}
+
+a.extLink[href^="http://pc.animelo.jp/"], a.extLink[href^="https://pc.animelo.jp/"]
+{
+	background: url('/Content/ExtIcons/animelo.png');
+}
+
+a.extLink[href^="http://pc.dwango.jp/"], a.extLink[href^="https://pc.dwango.jp/"]
+{
+	background: url('/Content/ExtIcons/animelo.png');
+}
+
+a.extLink[href^="http://www.beatport.com/"], a.extLink[href^="https://www.beatport.com/"]
+{
+	background: url('/Content/ExtIcons/beatport.png');
+}
+
+a.extLink[href^="http://www.e-onkyo.com/"], a.extLink[href^="https://www.e-onkyo.com/"],

```

Please remove the trailing comma.
```suggestion
a.extLink[href^="http://www.e-onkyo.com/"], a.extLink[href^="https://www.e-onkyo.com/"],
a.extLink[href^="http://www.e-onkyo.com/"], a.extLink[href^="https://www.e-onkyo.com/"]
```